league_table/__init__.py

```diff
@@ -1 +0,0 @@
-
```

Comment:
Why are we deleting this?

Reply:
My bad! Tim and I were discussing how to approach the league table with Hurumap. I thought I created the folder.